crates/subspace-farmer/src/single_plot_farm.rs

```diff
@@ -102,13 +103,15 @@ where
 /// Single plot farm abstraction is a container for everything necessary to plot/farm with a single
 /// disk plot.
 // TODO: Make fields private
+#[must_use = "Farm does not function properly unless run() method is called"]
 pub struct SinglePlotFarm {
     public_key: PublicKey,
     codec: SubspaceCodec,
     pub plot: Plot,
     pub commitments: Commitments,
```

Comment:
Should we hide those also, as we already have `SinglePlotPlotter`?

Reply:
We will, but benching code is very invasive and I wasn't able to remove it in this PR just yet, but as I introduce necessary abstractions or change benches it will certainly be hidden.